package.nls.es.json

```diff
@@ -34,38 +34,38 @@
+    "bookmarks.configuration.showCommandsInContextMenu.description": "Especifíca cómo se muestran los comandos en el menú contextual de Bookmark",
     "bookmarks.configuration.sidebar.expanded.description": "Especifica si la barra lateral se muestra expandida",
     "bookmarks.configuration.sideBar.countBadge.description": "Controla la insignia de conteo en el ícono de Bookmarks en la barra de actividad",
     "bookmarks.configuration.sideBar.countBadge.enumDescriptions.all": "Mostrar la suma de Bookmarks de todos los archivos",
     "bookmarks.configuration.sideBar.countBadge.enumDescriptions.files": "Mostrar la suma de archivos que contiene algunos Bookmarks",
     "bookmarks.configuration.sideBar.countBadge.enumDescriptions.off": "Deshabilitar la insignia de recuento de Bookmarks",
-    "bookmarks.configuration.multicursor.toggleMode.description": "Especifíca como se trata el modo multicursor cuando ya hay líneas con Marcadores",
+    "bookmarks.configuration.multicursor.toggleMode.description": "Especifíca cómo se trata el modo multicursor cuando ya hay líneas con Marcadores",
     "bookmarks.configuration.multicursor.toggleMode.enumDescriptions.allLinesAtOnce": "Crea un Marcador en todas las líneas a la vez si al menos una de las líneas no tiene ya un Marcador",
     "bookmarks.configuration.multicursor.toggleMode.enumDescriptions.eachLineIndependently": "Cambia el Marcador de cada Línea en vez de marcar todas igual",
-    "bookmarks.configuration.label.suggestion.description": "Especifica como se sugieren las etiquetas al crear un Marcador",
+    "bookmarks.configuration.label.suggestion.description": "Especifica cómo se sugieren las etiquetas al crear un Marcador",
     "bookmarks.configuration.label.suggestion.enumDescriptions.dontUse": "No utilizar la selección (Comportamiento Original)",
     "bookmarks.configuration.label.suggestion.enumDescriptions.useWhenSelected": "Usa el texto seleccionado directamente (si está disponible) y sin requerir confirmación",
     "bookmarks.configuration.label.suggestion.enumDescriptions.suggestWhenSelected": "Sugiere el texto seleccionado (si está disponible). Se sigue requiriendo confirmación.",
-    "bookmarks.configuration.label.suggestion.enumDescriptions.suggestWhenSelectedOrLineWhenNoSelected": "Sugiere el texto seleccionado (si esta disponible) o la línea entera (cuando no hay nada seleccionado). Se sigue requiriendo confirmación.",
+    "bookmarks.configuration.label.suggestion.enumDescriptions.suggestWhenSelectedOrLineWhenNoSelected": "Sugiere el texto seleccionado (si está disponible) o la línea entera (cuando no hay nada seleccionado). Se sigue requiriendo confirmación.",
     "bookmarks.colors.lineBackground.description": "Color de fondo de la línea con Marcador",
     "bookmarks.colors.lineBorder.description": "Color del borde de la línea con Marcador",
     "bookmarks.colors.overviewRuler.description": "Color general de fondo para todos los Marcadores"
```

Comment:
Just noticed you missed a few entries, related to the walkthrough. Without this, the left column in the Walkthrough is displayed in english

![image](https://github.com/alefragnani/vscode-bookmarks/assets/3781424/2b074b13-9681-4424-858e-af4c9aac226b)

These are the missing entries

```json
"bookmarks.commands.openSideBar.title": 
...and
"bookmarks.walkthroughs.title": 
"bookmarks.walkthroughs.description": 
"bookmarks.walkthroughs.toggle.title":
"bookmarks.walkthroughs.toggle.description":
"bookmarks.walkthroughs.navigateToBookmarks.title":
"bookmarks.walkthroughs.navigateToBookmarks.description":
"bookmarks.walkthroughs.defineLabelsForYourBookmarks.title":
"bookmarks.walkthroughs.defineLabelsForYourBookmarks.description":
"bookmarks.walkthroughs.exclusiveSideBar.title":
"bookmarks.walkthroughs.exclusiveSideBar.description":
"bookmarks.walkthroughs.workingWithRemotes.title":
"bookmarks.walkthroughs.workingWithRemotes.description":
"bookmarks.walkthroughs.customizingAppearance.title":
"bookmarks.walkthroughs.customizingAppearance.description":
```

Take a look at `package.nls.json` as reference.

Reply:
Hello, @alefragnani. Thank you for telling me. I just translated those entries and created a new commit.

Reply:
It worked perfectly. Thanks!